access/discord/Makefile

```diff
@@ -44,7 +44,6 @@ release: clean $(BINARY)
 	mkdir $(RELEASE_NAME)
 	cp -rf $(BINARY) \
 		README.md \
-		CHANGELOG.md \
```

Comment:
Any reason we do not want to include changelog with discord like with other plugins?

Reply:
It's empty on all the plugins I checked, the versions are wrong and it's not maintained as it's not part of our release process. I think we should remove them all.